src/main/java/com/yugabyte/sample/common/CmdLineOpts.java

```diff
@@ -34,793 +34,765 @@
 import com.yugabyte.sample.apps.*;
 
 /**
- * This is a helper class to parse the user specified command-line options if they were specified,
- * print help messages when running the app, etc.
+ * This is a helper class to parse the user specified command-line options if
```

Comment:
This file change seems off... may need to rebase and redo?

Reply:
Yes this was an eclipse formatting issue, have rebased